ChangeLog.md

```diff
@@ -9,11 +9,17 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
 ### Added
 - Added composable services for signature generation.
-- Support authoraide.
 
 ### Refactor
 - Refactored the signature generation to use composable services.
 
+## [v1.0.3] - 2024-04-07
+### Added
+- Rename author-aide to authoraide
+
+## [v1.0.3] - 2024-03-07
```

Comment:
I see this same tag [v1.0.3] in line no#16

Reply:
Hi Sreeni, this was added because it was same as when we tagged the older commit v1.0.3
https://github.com/Learnosity/learnosity-sdk-php/blob/v1.0.3/ChangeLog.md

Reply:
but I can change it in the next tag if you want